roles/edpm_update/tasks/packages.yml

```diff
@@ -0,0 +1,10 @@
+---
+
```

Comment:
Can we add the tasks to check for a running yum/dnf as done in https://github.com/openstack-archive/tripleo-ansible/blob/stable/wallaby/tripleo_ansible/roles/tripleo_packages/tasks/update.yml?